presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigDeployer.java

```diff
@@ -105,34 +119,43 @@ public void addConfigFiles(Path configDir, Path dataDir, Path pluginDir)
         }
     }
 
-    private static void createPropertiesFile(Properties properties, Path path, String comments)
-            throws PrestoManagerException
-    {
-        try (OutputStream output = newOutputStream(path, CREATE)) {
-            properties.store(output, comments);
-        }
-        catch (IOException e) {
-            throw new PrestoManagerException(format("Failed to create properties file: %s", path.getFileName()), e);
-        }
-    }
-
-    public void addConnectors(Path catalogDir)
+    /**
+     * Add default Presto connector configuration in the given directory
+     * <p>
+     * If {@link #defaultCatalog} exists, its contents will be used for
+     * the configuration directory. Otherwise, non-configurable default
+     * connector files will be created.
+     */
+    public void writeConnectorFiles(Path catalogDir)
```

Comment:
See my comment at `writeConfigFiles`, and the names of these two functions should be consistent.